airflow/lineage/__init__.py

```diff
@@ -45,6 +48,20 @@ class Metadata:
     data: Dict = attr.ib()
 
 
+def _get_backend() -> Optional[LineageBackend]:
+    _backend: Optional[LineageBackend] = None
+
+    try:
+        _backend_str = conf.get("lineage", "backend")
+        _backend = import_string(_backend_str)  # pylint:disable=protected-access
+    except ImportError as err:
+        log.debug("Cannot import %s due to %s", _backend_str, err)  # pylint:disable=protected-access
+    except AirflowConfigException:
+        log.debug("Could not find lineage backend key in config")
```

Comment:
Should there be a fallback to some default backend?

Reply:
Lineage is an optional feature, and this is just restoring the deleted code, so I don't think we need a default.

Reply:
By default there is no backend, lineage stays only in the XComs.